crates/algod_client/src/models/asset_params.rs

```diff
@@ -30,7 +30,7 @@ pub struct AssetParams {
     pub creator: String,
     /// \[dc\] The number of digits to use after the decimal point when displaying this asset. If 0, the asset is not divisible. If 1, the base unit of the asset is in tenths. If 2, the base unit of the asset is in hundredths, and so on. This value must be between 0 and 19 (inclusive).
     #[serde(rename = "decimals")]
-    pub decimals: u32,
+    pub decimals: u64,
```

Comment:
This is interesting, given it can fit within a u8.
Looks like everything is becoming a u64.